.github/workflows/_eng-tools-test.yaml

```diff
@@ -1,4 +1,5 @@
-name: Templates - eng/tools - Test
+# Prefix with "~" to sort last in Actions list
+name: ~Templates - eng/tools - Test
```

Comment:
I assume you have considered reusable workflows? https://docs.github.com/en/actions/using-workflows/reusing-workflows I don't know the trade-offs between these approaches, but I'd be curious.

Reply:
If you're asking "why prefix the name with `~Template`", it's because template (aka "reusable") workflows still appear in the "Actions" UI, even if they are only used as templates and never run directly.  I wanted to group/hide these in the UI as much as possible.

![image](https://github.com/Azure/azure-rest-api-specs/assets/9459391/852e81cb-9093-4ba0-99ea-07600f35cbe9)

Reply:
I guess I didn't expect a reusable workflow to show up on the actions page but I'm also not familiar with them enough to know.